source/setup-standard-clion-cmake.html.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Standard Configuration
+
+language_tabs:
+
+toc_footers:
+
+includes:
+
+search: true
+---
+
+# CLion CMake
+
+ 1. Once you've installed OpenSSL and cloned sleepy-discord into your project directory, open the project in CLion. 
```

Comment:
this is missing many steps, I think it's better not to include the previous steps.